src/enums/move-category.ts

```diff
@@ -1,5 +1,5 @@
 export enum MoveCategory {
+  STATUS = 1,
```

Comment:
NIT: We could alternatively add `UNKNOWN` or `UNSPECIFIED` group as `0` index.

EDIT: This would differ from the `Type` enum then though.. hm...